x-pack/plugins/security_solution/public/flyout/right/components/description.tsx

```diff
@@ -39,8 +48,32 @@ export interface DescriptionProps {
 export const Description: VFC<DescriptionProps> = ({ expanded = false }) => {
   const [isExpanded, setIsExpanded] = useState(expanded);
 
-  const { dataFormattedForFieldBrowser } = useRightPanelContext();
-  const { isAlert, ruleDescription } = useBasicDataFromDetailsData(dataFormattedForFieldBrowser);
+  const { dataFormattedForFieldBrowser, scopeId, eventId } = useRightPanelContext();
+  const { isAlert, ruleDescription, ruleId, ruleName } = useBasicDataFromDetailsData(
+    dataFormattedForFieldBrowser
+  );
+
+  const viewRule = useMemo(
+    () =>
+      !isEmpty(ruleName) && (
+        <EuiFlexItem grow={false}>
+          <RenderRuleName
```

Comment:
I was trying to figure out if there would be a slightly more elegant way to handle this, I feel like using an [button with icon](https://elastic.github.io/eui/#/navigation/button#icon-buttons) from Eui would prevent having to style the margin... but the `RenderRuleName` component isn't quite what we would want then...

One option that I see is to duplicate the `goToRuleDetails`, `href` and whatever those guys need from the [RenderRuleName component](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/timelines/components/timeline/body/renderers/formatted_field_helpers.tsx#L52) here and have our own button...

Or modify the RenderRuleName component to be more flexible...

But honestly I'm not sure, it's more work and might be actually that much cleaner... I'm fine with merging what you have now!